__tests__/petTrack.test.js

```diff
@@ -0,0 +1,188 @@
+
+// PROTOTYPE SYNTAX PET
+
+const Pet = require('../src/petTrack');
+
+describe('constructor', () => {
+  it('returns an object', () => {
+    expect(new Pet('Alfie')).toBeInstanceOf(Object);
+  });
+  it('sets the name property of pet', () => {
+    const pet = new Pet('Alfie')
+    expect(pet.name).toBe('Alfie')
+  });
+  it('has an intial age of 0', () => {
+    const pet = new Pet('Alfie')
+    expect(pet.age).toEqual(0)
+  });
+  });
+
+describe('growUp', () => {
+  it('throws en error if the pet is alive', () => {
```

Comment:
Typo. Theres a few of these where its been copy pasted so should go through and fix the others too
```
  it('throws an error if the pet is alive', () => {
```